src/components/productListing/ProductsContainer.js

```diff
@@ -10,29 +14,56 @@ const ProductsContainer = () => {
       try {
         let response = await axios.get("/api/products");
         setProducts(response.data.products);
+        console.log(response.data.products);
         setLoad(false);
       } catch (err) {
         console.log(err);
```

Comment:
You can destructure the data from  axios response.
```
const {data} = await axios.get(...)
```
 Always remove` console.log()  `from production build.

Also  add `setLoad(false)` of you received any error. 
You can also show the error in UI